addons/im_livechat/static/tests/helpers/mock_server/models/res_partner.js

```diff
@@ -15,10 +15,16 @@ patch(MockServer.prototype, {
             ["partner_id", "=", partner.id],
             ["is_pinned", "=", true],
         ]);
-        const livechats = this.getRecords("discuss.channel", [
+        const domain = [
             ["channel_type", "=", "livechat"],
             ["channel_member_ids", "in", members.map((member) => member.id)],
-        ]);
+        ];
+        // Livechat administrators have read access on all sessions.
+        const adminGroupId = this.getRecords("res.groups", [["name", "=", "livechat_admin"]]);
+        if (adminGroupId.length && this.pyEnv.currentUser.groups_id.includes(adminGroupId[0].id)) {
+            domain.pop();
+        }
+        const livechats = this.getRecords("discuss.channel", domain);
```

Comment:
The method hasn't changed in python, it shouldn't be changed here.

It makes no sense to return non-membership channels in this method, even if access is allowed.